.env.spec

```diff
@@ -1,4 +1,4 @@
 HTTP_SERVER_HOST=localhost
 
-CORS_ORIGIN=*
+# CORS_ORIGIN=*
```

Comment:
Would you mind adding a valid default CORS_ORIGIN in both `.env` files?

Reply:
I can do it, yet keeping this configuration on `.env` files makes the integration test to grab this value as config and by-pass cors testing. Do you want me to add them anyways?

Reply:
We can configure a suitable `CORS_ORIGN` for the integration tests and add a test that checks if it's working correctly by using a different origin. What do you think about it?

Reply:
The thing is that config component is built before being able to inject it on Integration Tests, not sure if we can mock/override the component to inject this variable while wiring the routes without modifying original initComponents function. Have you done this on WKC before?

Reply:
We don't have many integration tests yet, so we didn't have this issue before. We can add a test for it in the future if we're sure that it's working correctly in the development environments.